pmdk.spec

```
@@ -651,56 +649,62 @@ a device.
 
 %prep
 %autosetup -p1 -n %{name}-%{upstream_version}
-
+%if 0%{?rhel} == 7
+sed -i 's/cmake/cmake3/' src/deps/miniasync/Makefile
```

This could be a bit more defensive:

```suggestion
sed -i -e 's/cmake\([^3]\)/cmake3\1/' src/deps/miniasync/Makefile
```

So that it won't make any future mistake of changing `cmake3` to `cmake33` if upstream changes `cmake` to `cmake3`.